test/e2e/conformance_test.go

```diff
@@ -0,0 +1,205 @@
+// +build e2e
```

Comment:
This PR will also need the `/scripts/ci-e2e-conformance.sh` file from the release-0.4 branch to fully wire up the conformance test into prow